src/utl/src/LoggerCommon.cpp

```diff
@@ -52,28 +53,49 @@ void report(const char* msg)
   logger->report(msg);
 }
 
+namespace {
+std::string escapeCurlyBraces(const std::string& input) {
+    std::string output;
+    output.reserve(input.length());
+
+    for (char c : input) {
+        if (c == '{' || c == '}') {
+            output.push_back('X');
+        } else if (c == '[' || c == ']') {
+            output.push_back('Y');
+        } else {
+
+        output.push_back(c);
+        }
+    }
+
+std::cout << "xxxx " << output << std::endl; 
+    return output;
+}
+}
+
 void info(utl::ToolId tool, int id, const char* msg)
 {
   Logger* logger = getLogger();
-  logger->info(tool, id, msg);
+  logger->info(tool, id, escapeCurlyBraces(msg).c_str());
 }
 
 void warn(utl::ToolId tool, int id, const char* msg)
 {
   Logger* logger = getLogger();
-  logger->warn(tool, id, msg);
+  logger->warn(tool, id, escapeCurlyBraces(msg).c_str());
```

Comment:
This is not the preferred solution. 

Just change it to `logger->warn(tool, id, "{}", msg)`

Reply:
Same with info, error, and critical